mongoex/mongo_test.go

```diff
@@ -0,0 +1,23 @@
+package mongoex
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/assert/cmp"
+
+	"github.com/circleci/ex/testing/testcontext"
+)
+
+func TestMongoConnection(t *testing.T) {
+	ctx := testcontext.Background()
+	cfg := Config{
+		URI:    "mongodb://root:password@localhost:27107",
+		UseTLS: false,
+	}
+
+	client, err := New(ctx, "connection-test", cfg)
```

Comment:
We should close this client in a defer

Reply:
Isn't that the same client we're using to return the DB?

Reply:
The disconnect does that...

Reply:
Nope, I need to look at filenames better. Sorry 🤦‍♀️